src/routers/StackRouter.js

```diff
@@ -186,7 +186,11 @@ export default (
       }
 
       // Handle navigation to other child routers that are not yet pushed
-      if (action.type === NavigationActions.NAVIGATE) {
+      if (
+        action.type === NavigationActions.NAVIGATE &&
+        action.routeName !== 'DrawerOpen' &&
```

Comment:
@lorenzsell I don't have the context for the commit, so I am not sure, maybe @grabbou or @ericvicenti can shed some light.

The ideal fix will be in drawer navigator's code if it involves these route names.